rif/relay/index.md

```diff
@@ -5,10 +5,11 @@ tags: rif, relay, gas station network, gsn
 permalink: /rif/relay/
 ---
 
-RIF Relay is a secure sponsored transaction system that enables users to pay fees using ERC-20 tokens.
+RIF Relay is a secure sponsored transaction system that enables users to pay the transaction fees and gas consumption using ERC-20 tokens.
```

Comment:
```suggestion
RIF Relay is a secure sponsored transaction system that enables users to pay the transaction fees using ERC-20 tokens.
```
I'd remove "gas consumption" because it may lead the reader to think that the gas fees are paid in erc20 tokens, and that's not true since they're always paid by the worker in RBTC

Reply:
updated